### PR TITLE
Cards vue oh NO

### PR DESCRIPTION
**Pull request by chosta**

### CODE REVIEW

File: src/cards/base/NitrogenRichAsteroid.ts [hunk 1]
```diff
@@ -36,4 +38,16 @@ export class NitrogenRichAsteroid implements IProjectCard {
       }
       return game.increaseTemperature(player, 1);
     }
+
+    public metadata: CardMetadata = {
+      cardNumber: '037',
+      renderData: CardRenderer.builder((b) => {
+        b.productionBox((pb) => {
+          pb.plants(1).nbsp.or().br;
+          pb.plants(3).played.digit.colon().nbsp.plants(4).digit;
```
Comment (kberg) [inline src/cards/base/NitrogenRichAsteroid.ts:47]: Does this really look right? I'm tired, and it's late and not about to start up your UI, but hey, can you just visually check this please? :D

Comment (kberg) [reply src/cards/base/NitrogenRichAsteroid.ts:47]: I mean the iconographic distrinction between the plants on one side being plants and those on the other being plant tags.

File: src/cards/base/OlympusConference.ts [hunk 2]
```diff
@@ -51,4 +53,14 @@ export class OlympusConference implements IProjectCard, IResourceCard {
...
       return 1;
     }
+    public metadata: CardMetadata = {
+      cardNumber: '185',
+      renderData: CardRenderer.builder((b) => {
+        b.science().played.colon().science().br;
+        b.or().br;
+        b.minus().science().plus().cards(1);
+      }),
+      description: 'When you play a Science tag, including this, either add a Science resource to this card, or remove a Science resource from this card to draw a card.',
```
Comment (kberg) [inline src/cards/base/OlympusConference.ts:63]: Same, really. Which science() represnts a resource and which a tag? Is there a clear visual distinction?

**Review by kberg — APPROVED** [review]

File: src/cards/base/NitrogenRichAsteroid.ts (hunk 1 above)
Comment (chosta) [reply src/cards/base/NitrogenRichAsteroid.ts:47]: I've double checked it. No worries.

File: src/cards/base/OlympusConference.ts (hunk 2 above)
Comment (chosta) [reply src/cards/base/OlympusConference.ts:63]: Yes, there is. One is round the other one is squared

**Pull request merged**